src/Orleans.Core/Configuration/Options/LoadSheddingOptions.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+
+namespace Orleans.Hosting
+{
+    public class LoadSheddingOptions
+    {
+        /// <summary>
+        /// The LoadShedding element specifies the gateway load shedding configuration for the node.
+        /// If it does not appear, gateway load shedding is disabled.
```

Comment:
This should read something like, `Specified whether or not load gateway load shedding is enabled. The default value is false, meaning that gateway load shedding is disabled.`

Given this is about gateway load shedding, I wonder if we should name the class/props as such? Fine either way, we can change it in future.

Reply:
I mean that we don't have attributes or elements anymore, because this isn't XML config, and the notion of a value appearing or not is no longer applicable.

Reply:
Fixed.